recipes/pypandoc/build.sh

```diff
@@ -0,0 +1 @@
+$PYTHON setup.py install --single-version-externally-managed  --record record.txt
```

Comment:
@janschulz remove both `bld.bat` and `build.sh` and add the `script: python setup.py install --single-version-externally-managed  --record record.txt` in the build field of the yaml file.